chart/epinio/templates/default-user.yaml

```diff
@@ -22,6 +22,7 @@ metadata:
   namespace: {{ .Release.Namespace }}
 stringData:
   username: epinio
-  password: password
+  # bcrypt hash for 'password'
+  password: "$2a$10$6bCi5NMstMK781In7JGiL.B44pgoplUb330FQvm6mVXMppbXBPiXS"
```

Comment:
Not relevant to this PR but I just realized we always create this default user with the dummy `password`. Operators have no option to skip it, which would make perfect sense in production environments, where they would create users through other means. I created an issue here: https://github.com/epinio/helm-charts/issues/214